MagazineLayout/Public/MagazineLayout.swift

```diff
@@ -374,7 +374,15 @@ public final class MagazineLayout: UICollectionViewLayout {
         continue
       }
 
-      layoutAttributes.frame = headerFrame
+      var updatedHeaderFrame = headerFrame
+
+      if let refreshControl = currentCollectionView.refreshControl,
```

Comment:
Maybe `if currentCollectionView.refreshControl?.isRefreshing,`
instead of `if let refreshControl = currentCollectionView.refreshControl, refreshControl.isRefreshing`

Reply:
That would be a valid point if I wasn't using the height for the frame, otherwise I have to provide a default value of 0.